lib/buildingsync/model_articulation/loads_system.rb

```diff
@@ -114,39 +114,43 @@ def get_building_section(building_sections, standard_building_type, standard_spa
     end
 
     def adjust_people_schedule(space_type, building_section, model)
-      args = {}
-      args['hoo_per_week'] = building_section.typical_occupant_usage_value_hours
-
-      model_articulation_instance = OpenStudio::ModelArticulation::Extension.new
-      path = model_articulation_instance.measures_dir + '/create_parametric_schedules/measure.rb'
-      puts "create parametric schedule path: #{path}"
-      require path
-
-      # create an instance of the measure
-      measure = CreateParametricSchedules.new
-
-      # create an instance of a runner
-      runner = OpenStudio::Measure::OSRunner.new(OpenStudio::WorkflowJSON.new)
-
-      # get arguments
-      arguments = measure.arguments(model)
-      argument_map = OpenStudio::Ruleset.convertOSArgumentVectorToMap(arguments)
-
-      # populate argument with specified hash value if specified
-      arguments.each do |arg|
-        temp_arg_var = arg.clone
-        if args.key?(arg.name)
-          temp_arg_var.setValue(args[arg.name])
+      if !building_section.typical_occupant_usage_value_hours.nil?
+        args = {}
+        puts "building_section.typical_occupant_usage_value_hours: #{building_section.typical_occupant_usage_value_hours}"
+        args['hoo_per_week'] = building_section.typical_occupant_usage_value_hours
+
+        model_articulation_instance = OpenStudio::ModelArticulation::Extension.new
+        path = model_articulation_instance.measures_dir + '/create_parametric_schedules/measure.rb'
+        puts "create parametric schedule path: #{path}"
+        require path
```

Comment:
Requiring a measure here seems pretty bad form.  The better way to do this would be to factor out whatever code you are using from the measure into a method of the Model Articulation Gem then require it from there.

@davidgoldwasser FYI

Reply:
reformatted the measure, so we can call the code directly